Tests/Command/LoadDataFixturesDoctrineCommandTest.php

```diff
@@ -24,6 +25,14 @@ public function testInstantiatingWithoutManagerRegistry() : void
         try {
             new LoadDataFixturesDoctrineCommand($loader);
         } catch (TypeError $e) {
+            if (PHP_VERSION_ID >= 80000) {
+                $this->expectExceptionMessage(
```

Comment:
Why do we defined expected exception messages in a `catch` block ? That's weird.

Reply:
See the `expectedDeprecation` annotation on that method? I think this is why: in the case there is a deprecation, you don't want to expect an exception message at all.

Reply:
Can we keep one solution where only `$this->expectExceptionMessage()` is used instead of the annotation? It would improve readability.

Reply:
The annotation is for a deprecation, not for an exception.

Reply:
My bad, I misread it.

Reply:
@stof please review again